tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1357,21 +1357,18 @@ def remove_static_routes(standby_tor, active_tor_loopback_ip):
     standby_tor.shell('ip route del {}/32'.format(active_tor_loopback_ip), module_ignore_errors=True)
 
 
-def increase_linkmgrd_probe_interval(duthosts, tbinfo):
+def recover_linkmgrd_probe_interval(duthosts, tbinfo):
```

Comment:
Most of the code in this function duplicate with `update_linkmgrd_probe_interval`. Why not call `update_linkmgrd_probe_interval` in this function?

Reply:
Make sense, fixed